### PR TITLE
[ticket/15957] Disabling both choice

### PR DESCRIPTION
**Pull request by dhruveshk**
Disable both choice input when jabber is disabled in general settings

PHPBB3-15957

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15957

### CODE REVIEW

**Comment by dhruveshk** [comment]
I have changed the commit message why did the Travis CI build failed

**Comment by DavidIQ** [comment]
If you go through the details you should be able to determine why.

**Comment by dhruveshk** [comment]
I checked the job log, from that it is showing a curl error for facebook webdriver in postgres db build but I not change anything related to it.

**Comment by DavidIQ** [comment]
So sounds like a timeout. I'll try restarting it then.

File: phpBB/adm/style/acp_users_prefs.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
 		<dt><label for="notifymethod">{L_NOTIFY_METHOD}{L_COLON}</label><br /><span>{L_NOTIFY_METHOD_EXPLAIN}</span></dt>
 		<dd><label><input type="radio" class="radio" name="notifymethod" value="0"<!-- IF NOTIFY_EMAIL --> id="notifymethod" checked="checked"<!-- ENDIF --> /> {L_NOTIFY_METHOD_EMAIL}</label>
 			<label><input type="radio" class="radio" name="notifymethod" value="1"<!-- IF NOTIFY_IM --> id="notifymethod" checked="checked"<!-- ENDIF --><!-- IF S_JABBER_DISABLED --> disabled="disabled"<!-- ENDIF --> /> {L_NOTIFY_METHOD_IM}</label>
-			<label><input type="radio" class="radio" name="notifymethod" value="2"<!-- IF NOTIFY_BOTH --> id="notifymethod" checked="checked"<!-- ENDIF --> /> {L_NOTIFY_METHOD_BOTH}</label></dd>
+			<label><input type="radio" class="radio"  name="notifymethod" value="2"<!-- IF NOTIFY_BOTH --> id="notifymethod" checked="checked"<!-- ENDIF --><!-- IF S_JABBER_DISABLED --> disabled="disabled"<!-- ENDIF --> /> {L_NOTIFY_METHOD_BOTH}</label></dd>
```
Comment (marc1706) [inline phpBB/adm/style/acp_users_prefs.html]:
```suggestion
			<label><input type="radio" class="radio" name="notifymethod" value="2"<!-- IF NOTIFY_BOTH --> id="notifymethod" checked="checked"<!-- ENDIF --><!-- IF S_JABBER_DISABLED --> disabled="disabled"<!-- ENDIF --> /> {L_NOTIFY_METHOD_BOTH}</label></dd>
```

**Comment by marc1706** [comment]
Thanke you for your PR. I've take the liberty to adjust the minor issue I mentioned above and also fixed the commit author to correctly show you. You can do this in your local git installation as explained here: https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration

**Pull request merged**